### ISSUE TITLE
Customized context menu are cut off

### ISSUE THREAD

**Issue by GitHubStig** [handsontable/handsontable#2538]
We're using v0.13.1 but when we try to update to v0.15.b2, our customized context menu is cut off like so. Also tried v0.15.b6 which is still the same. Figured something along the way broke it.
![screen shot 2015-06-08 at 3 11 21 pm 2](https://cloud.githubusercontent.com/assets/9544391/8029575/5f0fd530-0df1-11e5-9abf-30a40c441e48.png)

To fix it 'temporary' to get it back to looking like this below
![screen shot 2015-06-08 at 3 10 33 pm 2](https://cloud.githubusercontent.com/assets/9544391/8029573/5a4a0b74-0df1-11e5-836c-74f484b89283.png)

we had to add this line of code
afterContextMenuShow: function() {
    $( '.htContextMenu' ).find( '.wtHolder' ).width( '235px' ).height( '408px' );
},

Hope this helps you find this bug.

**Comment by AMBudnik** [comment]
OK, thank you. Can you also include a demo of your code, in example [JSFiddle](https://jsfiddle.net/api/post/library/pure/)?

**Comment by GitHubStig** [comment]
Can't seem to get it working on JSFiddle...
```
<html>
	<head>
		<script src="http://code.jquery.com/jquery-2.1.3.min.js"></script>
		<link rel="stylesheet" media="screen" href="http://handsontable.com/dist/handsontable.full.css">
		<script src="http://handsontable.com/dist/handsontable.full.js"></script>
		<style>
			.htContextMenu table.htCore {
			  border: none;
			  outline: none;
			  border-radius: 6px;
			  box-shadow: 1px 1px 7px rgba(0,0,0,0.4);
			}
			
			.htContextMenu .wtSpreader {
			  top: 3px!important;
			  left: 3px!important;
			}
			
			.htContextMenu table tbody tr td {
			  padding: 7px 15px;
			}
			
			.htContextMenu table tbody tr:first-child td {
			 	border-radius: 6px 6px 0 0;
			}
			
			.htContextMenu table tbody tr:last-child td {
			  border-radius: 0 0 6px 6px;
			}
			
			.htContextMenu table.htCore {
			  width: 225px;
			}
			
			.htContextMenu table tbody tr td.current,
			.htContextMenu table tbody tr td.zeroclipboard-is-hover {
			  background: #e1ebed;
			}
			
			.htContextMenu table tbody tr td.htSeparator {
			  border-top: 1px solid #e1ebed;
			}
			
			.htContextMenu table tbody tr td.htDisabled, .htContextMenu table tbody tr td.htDisabled:hover {
			  color: #cfcfcf;
			}
		</style>
	</head>
	<body>
		<div id="example"></div>
		<script>
			var data = [
			        ["", "Ford", "Volvo", "Toyota", "Honda"],
			        ["2014", 10, 11, 12, 13],
			        ["2015", 20, 11, 14, 13],
			        ["2016", 30, 15, 12, 13]
			      ];
			      
			      var container = document.getElementById('example');
						var hot = new Handsontable( container, {
							data: data,
							width:527,
							height:480,
							minRows:25,
							minCols:10,
							rowHeaders:true,
							colHeaders:true,
							minSpareRows:3,
							minSpareCols:3,
			        contextMenu: {
							items: {
								'rowAbove': { name: 'Insert row above' },
								'rowBelow': { name: 'Insert row below' },
								'hsep1': '---------',
								'colLeft':  { name: 'Insert column on the left' },
								'colRight': { name: 'Insert column on the right' },
								'hsep2': '---------',
								'removeRow': { name: 'Remove row' },
								'removeCol': { name: 'Remove column' },
								'hsep3': '---------',
								'undo': { disabled: false },
								'redo': { disabled: false },
								'hsep4': '---------',
								'sortAsc':   { name: 'Sort column A -> Z' },
								'sortDesc':  { name: 'Sort column Z -> A' },
								'clearSort': { name: 'Clear sort order' },
							}
			        },        
						});
		</script>
	</body>
</html>
```

**Comment by AMBudnik** [comment]
I @GitHubStig did you find the cause of this behaviour? I was trying to create an example of yours and couldn't reproduce.

**Comment by GitHubStig** [comment]
Hi @AMBudnik,

Thanks for following up on this. At the moment, we are on the v0.19.0 and the context menu is no longer cropped but it overextends and looked like this below:

![screen shot 2016-02-11 at 4 19 18 pm](https://cloud.githubusercontent.com/assets/9544391/12971942/9a8b3d7c-d0db-11e5-83f3-7df5df9c5f11.png)
( It's grey all around because the handsontable is inside Bootstrap's modal dialog )

We're still using the same code snipper to fix the issue.

Today, I took another good look at it just now and I found out what's going on.

So what's happening was that back then our designer was styling the context menu using the .wtHider class and it looked fine till v0.15+ because .wtHider div container is way longer like this:

![screen shot 2016-02-11 at 4 12 44 pm 2](https://cloud.githubusercontent.com/assets/9544391/12972015/7e64439a-d0dc-11e5-9d41-cc9a0ccab3f9.png)

I guess the solution on our side is to style the context menu without using the .wtHider and there's nothing wrong with Handsontable.

Thanks again for bringing this up.
